spec/rails_helper.rb

```diff
@@ -94,9 +94,9 @@ def self.all_matches
 Sidekiq::Testing.fake!
 Sidekiq::Testing.server_middleware do |chain|
   chain.add Sidekiq::SemanticLogging
-  # chain.add Sidekiq::Instrument::ServerMiddleware
   chain.add Sidekiq::ErrorTag
 end
+require 'sidekiq/api'
```

Comment:
This seems to stand out -- not clear why it has to be in all the specs

Reply:
i was just putting it with the other `require 'sidekiq/testing'` but i can put it in the only test that uses it